kairon/events/server.py

```
@@ -144,3 +145,9 @@
 def dispatch_scheduled_event(event_id: Text = Path(description="Event id")):
     KScheduler().dispatch_event(event_id)
     return {"data": None, "message": "Scheduled event dispatch!"}
+
+
+@app.get('/api/mail/schedule/{bot}', response_model=Response)
+def request_epoch(bot: Text = Path(description="Bot id")):
```

Ruff (0.8.0)

Do not perform function call `Path` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)